astro_2.0/features/Bounties/components/BountiesFeed/BountiesFeed.module.scss

```diff
@@ -1,4 +1,57 @@
+@use 'styles/breakpoints';
+
 .root {
+  align-items: flex-start;
   display: flex;
   flex-direction: column;
+  justify-content: flex-start;
+  margin: 0 auto;
+  width: 100%;
+}
+
+.row {
+  align-items: flex-start;
+  display: flex;
+  justify-content: flex-start;
+  margin: 0 auto;
+  width: 100%;
+
+  @media screen and (max-width: #{breakpoints.$screen-lg}px) {
+    flex-direction: column;
+    padding: 0;
+  }
+}
+
+.additionalFilters {
+  justify-content: right;
+}
+
+.content {
+  margin-top: 24px;
+}
+
+.container {
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+}
+
+.bountyInfo {
```

Comment:
Looks like this style is not being used.

Reply:
fixed